packages/grafana-eslint-config/test.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+const {expect} = require('chai');
```

Comment:
This looks like prettier is not formatting, need to update the root package.json prettier commands to also run on js files :)

Reply:
Yeah, that's just something temporary that I'd created quickly to outline my plans.